test/source/npm_test.rb

```diff
@@ -54,6 +54,13 @@
         end
       end
 
+      it "does not include ignored dependencies" do
+        Dir.chdir fixtures do
+          @config.ignore({ type: "npm", name: "autoprefixer" })
```

Comment:
`@config.ignore` is expecting string keys `"type" => Licensed::Source::NPM.type, "name" => "autoprefixer"`